### PR TITLE
add types definitions

### PR DESCRIPTION
**Pull request by j8seangel**
Coming from https://github.com/DefinitelyTyped/DefinitelyTyped/pull/67548#issuecomment-1832702768 this would allow typescript users to consume the library without any extra type definitions

### CODE REVIEW

**Comment by andrewbranch** [comment]
To clarify, this only impacts users who are using old, soon-to-be deprecated module resolution settings, but it’s still best practice to help them out, because there are quite a few users still on it. Anyone using `--moduleResolution node16`, `nodenext`, or `bundler` is already good: https://arethetypeswrong.github.io/?p=geo-coordinates-parser%401.6.3

File: package.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "description": "A Javascript function for reading a variety of coordinate formats and converting to decimal numbers. Builds on other efforts by returning the verbatim coordinates and the decimal coordinates all in one object.",
   "type": "module",
   "main": "merge.js",
+  "types": "merge.d.ts",
```
Comment (andrewbranch) [inline package.json:7]:
```suggestion
  "main": "./dist/cjs/merge.js",
  "types": "./dist/cjs/merge.d.ts",
```

Neither `main` nor the proposed `types` point to existing files. In the spirit of this PR adding maximal backward compatibility, this fixes things both for old TS versions and old Node.js / bundler versions.

**Comment by andrewbranch** [comment]
FYI this PR didn’t fix anything; it needed to include my suggestion above to have any effect 🙂

**Comment by j8seangel** [comment]
Thanks @andrewbranch I applied your changes [here](https://github.com/ianengelbrecht/geo-coordinates-parser/pull/13) 🙇

**Pull request merged**